config/playground/eks/dev/README.md

```diff
@@ -0,0 +1,52 @@
+# Terraform - Elastic Kubernetes Getting Started Guide
```

Comment:
better to change the doc title to fit the actual usage of this doc

Reply:
fixed. thanks